Doc/library/io.rst

```diff
@@ -401,29 +401,29 @@ I/O Base Classes
       hint.
 
       Note that it's already possible to iterate on file objects using ``for
-      line in file: ...`` without calling ``file.readlines()``.
+      line in file: ...`` without calling :meth:`!file.readlines`.
 
    .. method:: seek(offset, whence=SEEK_SET, /)
 
       Change the stream position to the given byte *offset*.  *offset* is
       interpreted relative to the position indicated by *whence*.  The default
-      value for *whence* is :data:`SEEK_SET`.  Values for *whence* are:
+      value for *whence* is :data:`!SEEK_SET`.  Values for *whence* are:
 
-      * :data:`SEEK_SET` or ``0`` -- start of the stream (the default);
+      * :data:`!SEEK_SET` or ``0`` -- start of the stream (the default);
         *offset* should be zero or positive
-      * :data:`SEEK_CUR` or ``1`` -- current stream position; *offset* may
+      * :data:`!SEEK_CUR` or ``1`` -- current stream position; *offset* may
         be negative
-      * :data:`SEEK_END` or ``2`` -- end of the stream; *offset* is usually
+      * :data:`!SEEK_END` or ``2`` -- end of the stream; *offset* is usually
         negative
 
       Return the new absolute position.
 
       .. versionadded:: 3.1
-         The ``SEEK_*`` constants.
+         The :data:`!SEEK_*` constants.
 
       .. versionadded:: 3.3
          Some operating systems could support additional values, like
-         :const:`os.SEEK_HOLE` or :const:`os.SEEK_DATA`. The valid values
+         :const:`!SEEK_HOLE` or :const:`!SEEK_DATA`. The valid values
```

Comment:
We should document these constants rather than suppressing the cross-reference (we could add them to ``nitpick_ignore`` in this PR, though.

Reply:
Yes, I plan to do so. But, we need to add them first. Currently, they are not exposed at all.

Reply:
> Currently, they are not exposed at all.

Actually, they _are_ exposed in `os` via `posix`.

Reply:
See #107936